src/renderer/components/Editor/TranscriptionBlock.tsx

```diff
@@ -45,14 +48,36 @@ const TranscriptionBlock = ({
     (store: ApplicationStore) => store.selection
   );
 
-  const selectionSet = useMemo(() => new Set(selectionArray), [selectionArray]);
+  const editWord = useSelector((store: ApplicationStore) => store.editWord);
+
+  const selectionSet = useMemo(
+    () => (editWord === null ? new Set(selectionArray) : new Set()),
+    [selectionArray, editWord]
+  );
 
   const dispatch = useDispatch();
 
+  const submitWordEdit: () => void = () => {
+    if (
+      editWord !== null &&
+      editWord.text !== '' &&
```

Comment:
Not sure if this is expected behaviour (so correct me if I'm wrong), but currently when a word is double clicked to be edited, and then the entire word is deleted via backspace button, pressing enter brings back the entire word. I would assume if a user manually deletes the word (even though we have deleting as a feature), they don't want the word to be there.

Reply:
It’s expected, because the default behaviour without this safety check is that the word’s text just becomes “” and the word takes up space without being able to be deleted, selected or edited.

i like your idea better though - why not just let this be an alternative way of deleting the word! I’ll implement that